spec/functions/dns_soa_spec.rb

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env ruby -S rspec
```

Comment:
This is not a valid shebang.  On Debian:

```
/usr/bin/env: 'ruby -S rspec': No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

Of course, the `-S` flag of `env(1)` is not portable: on FreeBSD:

```
env: ruby -S rspec: No such file or directory
```

So better remove the shebang (and make sure the file is not executable).

Reply:
Also, this file can be removed because there is no such "legacy" function :smile: